docs/cloud-native-security/cspm-cloud-posture-dashboard.asciidoc

```diff
@@ -26,14 +26,14 @@ The Cloud Security Posture dashboard shows:
 
 At the top of the dashboard, you can switch between the Cloud accounts and Kubernetes cluster views.
 
-The top section of either view summarizes your overall cloud security posture (CSP) by aggregating data from all monitored resources. Three summary cards on the left show Cloud Accounts or Clusters evaluated, Resources evaluated, and Failed Findings. You can click *Resources Evaluated* or *Failed Findings* to open the <<findings-page, Findings page>>, grouped by resources or filtered for failed findings, respectively.
+The top section of either view summarizes your overall cloud security posture (CSP) by aggregating data from all monitored resources. The summary cards on the left show the number of Cloud Accounts or Clusters evaluated, and the number of Resources evaluated. You can click *Enroll more accounts* or *Enroll more clusters* to deploy to additional cloud assets. 
```

Comment:
It looks like we have varying capitalization of 'cloud account(s)' throughout this page ('Cloud accounts', 'cloud accounts', and here, 'Cloud Accounts'). I'm not sure which is correct, but we should probably standardize this for consistency.
I think also in this context we can probably lowercase 'Clusters' and 'Resources', since we're not quoting UI copy, and these are lowercase elsewhere on the page.